packages/ui/src/components/Lessons/ManageLesson/Confirmation.tsx

```diff
@@ -20,6 +21,7 @@ export const Confirmation: React.FC<{
   onConfrim: Void;
   onEdit: Void;
   confirmationLoading?: boolean;
+  isLargeScreen: boolean;
```

Comment:
No need for this, you can use media query directlly here. This component is not using anywhere else other than the web workspace.